src/components/core/pivotTable/tests/agGridApiWrapper.spec.tsx

```diff
@@ -0,0 +1,55 @@
+// (C) 2007-2019 GoodData Corporation
+import * as React from 'react';
+import { mount } from 'enzyme';
+import { AgGridReact } from 'ag-grid-react';
+
+import ApiWrapper from '../agGridApiWrapper';
+import { GridReadyEvent, GridApi } from 'ag-grid';
+import { ICustomGridOptions } from '../../PivotTable';
+
+describe('agGridApiWrapper', () => {
+    function renderComponent(customProps = {}) {
+        const gridOptions: ICustomGridOptions = {
+            rowModelType: 'infinite'
+        };
+
+        const defaultProps = { gridOptions };
+
+        return mount(
+            <AgGridReact
+                {...defaultProps}
+                {...customProps}
+            />
+        );
+    }
+
+    describe('getHeaderHeight()', () => {
```

Comment:
`getHeaderHeight() -> getHeaderHeight` this will link test describe with function definition in idea